framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1124,6 +1125,13 @@ private AnnotatedExecutableType(ExecutableType type, AnnotatedTypeFactory factor
         private boolean paramTypesComputed = false;
         /** The receiver type. */
         private AnnotatedDeclaredType receiverType;
+        /**
+         * The varargs type is the last element of {@link paramTypes} if the method or constructor
+         * accepts a variable number of arguments and the {@link paramTypes} has not been adapted
+         * yet. This type needs to be stored in the field to avoid being affected by {@link
```

Comment:
Can you add a reference to what "adapted" means here? Is it viewpoint adaptation? Or something else?
Make the relationship to the parameter types clearer. What are they after adaptation?